playbooks/ansible-test-base/pre.yaml

```diff
@@ -54,10 +54,13 @@
       shell: ~/venv/bin/pip install selinux "setuptools<50.0.0"
       when: ansible_os_family == "RedHat"
 
-    # ara is installing ansible-2.10, when 2.9 is needed. Commenting to temporarily fix the issue.
-    # - name: Install ara into virtualenv
-    #   shell: ~/venv/bin/pip install "ara<1.0.0"
-    #   when: "'integration' in ansible_test_command"
+    - name: Install ara into virtualenv
+      shell: ~/venv/bin/pip install "ara<1.0.0"
+      when: "'integration' in ansible_test_command"
+    
+    - name: Uninstall ara installed ansible(s)
```

Comment:
This just uninstalls the ansible that ara pulls in, so that we have the correct one based on the actual integration job. But totally agree about the moving to ara 1.0+. I only attempted to get things up and running until we have it setup. Moving to ara 1.0+ would also mean skipping the report generation for Py2.7 jobs.

Reply:
Yes, I understand. I thought we discussed moving directly to 1.0+ moving forward.

For 2.7 integration jobs, that is fine as we will be only running py3.8 shortly.